Project.toml

```diff
@@ -11,7 +11,7 @@ Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 
 [compat]
 HTTP = "0.9"
-PProf = "2"
+PProf = "2.1"
```

Comment:
`"2"` should already be okay. It is short hand for `[2.0.0 , 3.0.0)` - i.e. anything that's `2.*` would match.

Whereas `2.1` would exclude version `2.0`.

Reply:
It seems that we do want to exclude 2.0, because it doesn’t have alloc decoding

Reply:
Ah yeah, you're right. Makes sense. It's because I was still thinking we'd want to support both the presence and absence of this feature. Otherwise we could get any updates we make to this package in our 1.6 RAI code base until we upgraded all the way to 1.8!

So we want to support both. It should be easy to do that, I'll have a look at this tomorrow

Reply:
k, i think we've addressed this now. deleting this again. :)